chaincfg/params.go

```diff
@@ -687,3 +742,8 @@ func init() {
 	mustRegister(&RegressionNetParams)
 	mustRegister(&SimNetParams)
 }
+
+func createPow(value string) *big.Int {
```

Comment:
Better name it something like "powTargetFromString"

Reply:
Renamed